src/main/webapp/WEB-INF/web.xml

```diff
@@ -36,23 +36,33 @@
     <servlet-class>com.google.codeu.servlets.MessageServlet</servlet-class>
   </servlet>
 
+  <servlet>
+    <servlet-name>aboutme</servlet-name>
+    <servlet-class>com.google.codeu.servlets.AboutMeServlet</servlet-class>
+  </servlet>
+
   <servlet-mapping>
-    <url-pattern>/login</url-pattern>
```

Comment:
What's with this reordering here?

I think it makes more sense to specify 'url-pattern' first, because from/source logically precedes to/destination...

That is, I think it is easier to read this as:

    "When you see the URL /foo, then route it to servlet bar."

Specifying the servlet name, first, rather than the URL pattern first creates a more awkward sentence for reading/understanding what the stanza does.

If you feel really strongly that it should be ordered the other way, I'm fine with that, but pull requests should be logically focused; I'd ask that if you want to reorder that you make such a change in a separate clean-up/maintenance pull request focused on this cosmetic change rather than mixing unrelated cosmetic changes into your feature change. Is that OK with you?

Reply:
It's okay with me. I rearranged them to understand the logic and implement the changes.